src/foam/blob/Blob.js

```diff
@@ -396,6 +392,9 @@ foam.CLASS({
             type: 'Object',
           }
         ],
+      code: function compareTo(other) {
+        return foam.blob.IdentifiedBlob.isInstance(other) && other.id == this.id;
```

Comment:
How can this be correct? compareTo should return -1, 0, or 1, but this is returning a boolean.